src/components/dashboard/DashboardContainer.js

```diff
@@ -1,22 +1,10 @@
 import React, {Component, PropTypes} from 'react';
 import {connect} from 'react-redux';
 
-import {getUserInfo} from '../../actions/userActions';
-import {getRoaster} from '../../actions/roasterActions';
 import Dashboard from './Dashboard';
 
 class DashboardContainer extends Component {
 
-    componentDidMount() {
```

Comment:
I added this last time, but we don't need it anymore. It was causing problems anyway.